.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -255,10 +255,12 @@ on:
           - eltwise.ternary.lerp
           - eltwise.ternary.where.where
           - eltwise.ternary.where.where_pytorch2
+          - eltwise.ternary_backward.addcmul_bw
```

Comment:
Did you miss these from a prior PR? Did the post-commit not catch this?

Reply:
Just moved this up to be near other ternary_backward ops.